Automattic/wp-calypso
# Site Settings: Move Press This in a separate component

This PR moves **Press This** in a separate component. In addition to separating concerns and reducing the size of `form-writing.jsx` and the main settings stylesheet, this is necessary, because as specified in p6TEKc-HQ-p2, **Press This** will be part of another card for Jetpack sites. So, while this PR does not change anything visual, it prepares the Press This block for being used within another card.

To test:
* Checkout this branch
* Go to `/settings/writing/$site` for a WordPress.com site and verify there are no visual changes or regressions.
* Go to `/settings/writing/$site` for a Jetpack site and verify there are no visual changes or regressions.
* Verify the Press This bookmarklet works as it did before.
